mapswipe_workers/mapswipe_workers/firebase_to_postgres/transfer_results.py

```diff
@@ -92,9 +93,18 @@ def results_to_file(results, projectId):
     logger.info(f'Got %s groups for project {projectId} to transfer' % len(results.items()))
     for groupId, users in results.items():
         for userId, results in users.items():
-            timestamp = results['timestamp']
-            start_time = results['startTime']
-            end_time = results['endTime']
+
+            # check if all attributes are set, if not don't transfer the results for this group
+            try:
+                timestamp = results['timestamp']
+                start_time = results['startTime']
+                end_time = results['endTime']
+                results = results['results']
+            except KeyError as e:
+                sentry.capture_exception_sentry(e)
```

Comment:
are the logs going to be available? (do they get exported from the docker container, etc...) if not, you could use sentry's message capture https://docs.sentry.io/error-reporting/capturing/?platform=python#capturing-messages to keep a record of what group/project/user this happened in.

Reply:
a good idea to have this in sentry as well, will make it possible for us to track by just looking at sentry then.